internal/source/git.go

```diff
@@ -54,7 +56,7 @@ func (r *Git) Unpack(ctx context.Context, bundle *rukpakv1alpha1.Bundle) (*Resul
 	// Clone
 	repo, err := git.CloneContext(ctx, memory.NewStorage(), memfs.New(), &cloneOpts)
 	if err != nil {
-		return nil, fmt.Errorf("bundle unpack git clone error: %w", err)
+		return nil, fmt.Errorf("bundle unpack git clone error: %w - %s", err, progress.String())
```

Comment:
Would you mind including the resulting output of a clone failure log message so we can see how this looks?

Without having looked at it myself, I wonder if it would make sense to use a new line to allow the git clone output to begin on its own line?

```suggestion
		return nil, fmt.Errorf("bundle unpack git clone error: %w\n%s", err, progress.String())
```

But maybe my suggestion looks worse because the default logger encodes this error string such that the log actually ends up with "\n" in it?

Reply:
@joelanford You are right.  The log message gets "\n" in it.  I leave it as-is.